docs/Account/methods.md

```diff
@@ -238,7 +239,31 @@ const { transactionHash, userOperationReceipt } = await wait();
 
 **Returns**
 
-- userOpResponse (`Promise<UserOpResponse>`): userOpResponse that you can use to track user operation.
+- userOpResponse (`Promise<UserOpResponse>`): The method returns an object of type `UserOpResponse` which has a `userOpHash` and two methods: `wait()` and `waitForTxHash()`.
+
+  ```ts
+  type UserOpResponse = {
+    userOpHash: string;
+    wait(_confirmations?: number): Promise<UserOpReceipt>;
+    waitForTxHash(): Promise<UserOpStatus>;
+  };
+
+  type UserOpReceipt = {
+    userOpHash: string;
+    entryPoint: string;
+    sender: string;
+    nonce: number;
+    paymaster: string;
+    actualGasCost: BigNumber;
```

Comment:
@joepegler can you confirm.

Reply:
```
 type UserOpReceipt = {
  /* The request hash of the UserOperation. */
  userOpHash: string;
  /* The entry point address used for the UserOperation. */
  entryPoint: string;
  /* The paymaster used for this UserOperation (or empty). */
  paymaster: string;
  /* The actual amount paid (by account or paymaster) for this UserOperation. */
  actualGasCost: Hex;
  /* The total gas used by this UserOperation (including preVerification, creation, validation, and execution). */
  actualGasUsed: Hex;
  /* Indicates whether the execution completed without reverting. */
  success: "true" | "false";
  /* In case of revert, this is the revert reason. */
  reason: string;
  /* The logs generated by this UserOperation (not including logs of other UserOperations in the same bundle). */
  logs: Array<any>; // The logs generated by this UserOperation (not including logs of other UserOperations in the same bundle)
  /* The TransactionReceipt object for the entire bundle, not only for this UserOperation. */
  receipt: any;
};
``` This is how the type in SDK, please update it in docs @himanshugarg06